spec/models/state_file_w2_spec.rb

```diff
@@ -71,10 +71,10 @@
     end
 
     context "states where we don't show local boxes" do
-      let(:intake) { create :state_file_nc_intake }
+      let(:intake) { create :state_file_az_intake }
```

Comment:
why did this change from nc to az? both of them do not have local boxes shown, right?

Reply:
Yes, this is a no-op change and I just wanted to double-check that there wasn't something weird about AZ, since that was the first acceptance failure I heard about on the previous PR.